src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/taskconfiguration/services/CaseConfigurationProviderServiceTest.java

```diff
@@ -156,4 +160,232 @@ void gets_fields_to_map() {
+        String someCaseId = "someCaseId";
+
+        when(ccdDataService.getCaseData(someCaseId)).thenReturn(caseDetails);
+        when(caseDetails.getData()).thenReturn(Map.of("caseAccessCategory", "categoryA"));
+        List<PermissionsDmnEvaluationResponse> permissions = asList(
+            new PermissionsDmnEvaluationResponse(
+                stringValue("tribunalCaseworker"),
+                stringValue("Read,Refer,Own,Manage,Cancel"),
+                null,
+                null,
+                null,
+                stringValue("LEGAL_OPERATIONS"),
+                stringValue("categoryB")
+            ),
+            new PermissionsDmnEvaluationResponse(
+                stringValue("seniorTribunalCaseworker"),
+                stringValue("Read,Refer,Own,Manage,Cancel"),
+                null,
+                null,
+                null,
+                stringValue("LEGAL_OPERATIONS"),
+                stringValue("categoryC")
+            )
+        );
+
+        String caseData = "{\"caseAccessCategory\":\"categoryA\"}";
+        when(dmnEvaluationService.evaluateTaskPermissionsDmn("IA", "Asylum", caseData, "{}"))
+            .thenReturn(permissions);
+
+
+        TaskConfigurationResults mappedData = caseConfigurationProviderService
+            .getCaseRelatedConfiguration(someCaseId, Map.of());
+
+        Assertions.assertThat(mappedData.getPermissionsDmnResponse()).isEmpty();
+    }
+
+    @Test
+    void should_consider_permissions_when_case_access_category_column_is_present_but_case_field_is_null() {
+        String someCaseId = "someCaseId";
+
+        when(ccdDataService.getCaseData(someCaseId)).thenReturn(caseDetails);
+        List<PermissionsDmnEvaluationResponse> permissions = asList(
+            new PermissionsDmnEvaluationResponse(
+                stringValue("tribunalCaseworker"),
+                stringValue("Read,Refer,Own,Manage,Cancel"),
+                null,
+                null,
+                null,
+                stringValue("LEGAL_OPERATIONS"),
+                stringValue("categoryB")
+            ),
+            new PermissionsDmnEvaluationResponse(
+                stringValue("seniorTribunalCaseworker"),
+                stringValue("Read,Refer,Own,Manage,Cancel"),
+                null,
+                null,
+                null,
+                stringValue("LEGAL_OPERATIONS"),
+                stringValue("categoryC")
+            )
+        );
+
+        when(dmnEvaluationService.evaluateTaskPermissionsDmn("IA", "Asylum", "{}", "{}"))
+            .thenReturn(permissions);
+
+
+        TaskConfigurationResults mappedData = caseConfigurationProviderService
+            .getCaseRelatedConfiguration(someCaseId, Map.of());
+
+        Assertions.assertThat(mappedData.getPermissionsDmnResponse()).isEmpty();
+    }
+
+    @Test
+    void should_consider_permissions_when_case_access_category_column_is_present_but_case_field_is_empty() {
```

Comment:
can we rename this to match the behaviour and outcome? e.g should_return_empty_when.........